### PR TITLE
[incubator/cassandra] fix custom selector issue

### PR DESCRIPTION
**Pull request by yisiqi**
Signed-off-by: Yi Siqi <yisq@me.com>

#### What this PR does / why we need it:
Fix the issue that charts of cassandra can not render custom selector rightly.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

File: incubator/cassandra/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: cassandra
-version: 0.6.2
+version: 0.6.3
```
Comment (maver1ck) [inline incubator/cassandra/Chart.yaml:2]: @yisiqi Could you please bump version to 0.6.4 ?

**Review by maver1ck — COMMENTED** [review]
/assign

File: incubator/cassandra/templates/statefulset.yaml [hunk 2]
```diff
@@ -8,6 +8,13 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+  selector:
+    matchLabels:
+      release: "{{ .Release.Name }}"
+      app: {{ template "cassandra.name" . }}
+{{- if .Values.selector }}
```
Comment (maver1ck) [inline incubator/cassandra/templates/statefulset.yaml:15]: I think ideas of Values.selector was to use it as ability to create nodeSelector.
So place in pod spec is ok and we shouldn't change this.
Could you please look here ?  https://github.com/helm/charts/pull/8320/files
I think it's enough to fix this chart
PS. I'm trying to get rights to merge this fix. I hope I will do it next week.

**Review by maver1ck — COMMENTED** [review]
/assign

**Comment by maver1ck** [comment]
PS. I closed this bug by https://github.com/helm/charts/pull/8320

**Comment by davidkarlsen** [comment]
Fixed with  #8320

**Pull request closed without merging**